sahi/model.py

```diff
@@ -66,7 +65,7 @@ def __init__(
         self._object_prediction_list_per_image = None
 
         # automatically set device if its None
-        if not (self.device):
+        if not (self.set_device):
             self.device = "cuda:0" if is_torch_cuda_available() else "cpu"
```

Comment:
bu satiri set_device fonksiyonu icerisine tasiip fonksiyonu cagiracaksin:
```suggestion
            self.set_device()
```

Reply:
@fcakyon Buradaki önerinizi pek anlayamadım.
Bu kodu ```self.device = "cuda:0" if is_torch_cuda_available() else "cpu"``` bununla mı değiştirmem gerekiyor?

```self.set_device = ""cuda:0" if is_torch_cuda_available() else "cpu"```

https://github.com/kadirnar/sahi/blob/tfhub/sahi/model.py#L102 ve bunu torch için düzenlemem gerekmiyor mu?

Reply:
burada sadece set_device fonksiyonu cagirilmali, set_device fonksiyonu zaten set ediyor device'i. tekrardan ayrica device set etmemen gerekiyor.